app/Resources/install/dataimport/SOM_competency_data.sql

```diff
@@ -0,0 +1,34 @@
+LOCK TABLES `competency` WRITE;
```

Comment:
good catch. they are left over from an earlier, aborted attempt to run queries against the db via doctrine.
removing them...